include/netutils/webclient.h

```diff
@@ -191,13 +191,17 @@ struct webclient_context
    *   url          - A pointer to a string containing the full URL.
    *                  (e.g., http://www.nutt.org/index.html, or
    *                   http://192.168.23.1:80/index.html)
+   *   unix_socket  - If not NULL, the path to an AF_LOCAL socket.
    *   headers      - An array of pointers to the extra headers.
    *   nheaders     - The number of elements in the "headers" array.
    *   bodylen      - The size of the request body.
    */
 
   FAR const char *method;
   FAR const char *url;
```

Comment:
why not reuse url for unix_socket?

Reply:
because url is still used for the http request.

Reply:
so how do you handle both url and unix_socket not NULL?

Reply:
the url contains hostname, path, query string, etc.
they will be sent over the unix domain socket.
(while the hostname is less meaningful for unix domain, we need to send something anyway.)